[SQA] Issue while building the all cluster app in Thread setup

**Issue summary**:  Not able to build all cluster app in Thread setup

**Code used**: `west build -b nrf52840dk_nrf52840 --pristine always -- -DCONFIG_CHIP_LIB_SHELL=y` in examples/all-clusters-app/nrfconnect folder

**Commit used**: 5e4a98cbb9362eee073e6e58e9ea0d6032112146

**Error log**: 

> ﻿build@e835fed4cb84:/var/chip/examples/all-clusters-app/nrfconnect$ west build -b nrf52840dk_nrf52840 --pristine always -- -DCONFIG_CHIP_LIB_SHELL=y
-- west build: generating a build system
-- Found Python3: /usr/bin/python3.9 (found version "3.9.5") found components: Interpreter 
Including boilerplate (Zephyr base): /var/ncs/zephyr/cmake/app/boilerplate.cmake
-- Application: /var/chip/examples/all-clusters-app/nrfconnect
-- Zephyr version: 2.7.99 (/var/ncs/zephyr), build: v2.7.99-ncs1
-- Found Python3: /usr/bin/python3.9 (found suitable exact version "3.9.5") found components: Interpreter 
-- Found west (found suitable version "0.12.0", minimum required is "0.7.1")
-- Board: nrf52840dk_nrf52840
-- Cache files will be written to: /var/ncs/zephyr/.cache
-- Found dtc: /usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6")
-- Found toolchain: gnuarmemb (/opt/gcc-arm-none-eabi-9-2019-q4-major)
-- Found BOARD.dts: /var/ncs/zephyr/boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts
-- Found devicetree overlay: /var/chip/examples/all-clusters-app/nrfconnect/boards/nrf52840dk_nrf52840.overlay
-- Generated zephyr.dts: /var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/zephyr.dts
-- Generated devicetree_unfixed.h: /var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/include/generated/devicetree_unfixed.h
-- Generated device_extern.h: /var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/include/generated/device_extern.h
-- Including generated dts.cmake file: /var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/dts.cmake
warning: Experimental symbol CHIP is enabled.
Parsing /var/chip/examples/all-clusters-app/nrfconnect/Kconfig
Loaded configuration '/var/ncs/zephyr/boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840_defconfig'
Merged configuration '/var/chip/config/nrfconnect/app/sample-defaults.conf'
Merged configuration '/var/chip/examples/all-clusters-app/nrfconnect/prj.conf'
Merged configuration '/var/chip/examples/all-clusters-app/nrfconnect/boards/nrf52840dk_nrf52840.conf'
Merged configuration '/var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/misc/generated/extra_kconfig_options.conf'
Configuration saved to '/var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/.config'
Kconfig header saved to '/var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/include/generated/autoconf.h'
/opt/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gdb: error while loading shared libraries: libncurses.so.5: cannot open shared object file: No such file or directory
-- The C compiler identification is GNU 9.2.1
-- The CXX compiler identification is GNU 9.2.1
-- The ASM compiler identification is GNU
-- Found assembler: /opt/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc
-- OpenThread Source Directory: /var/ncs/modules/lib/openthread
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Searching 16 bit integer - Using unsigned short
-- Check if the system is big endian - little endian
-- Package Name: OPENTHREAD
-- Setting default package version: thread-reference-20200818-ncs1-rc3-560-g02e61a2ed
-- Package Version: thread-reference-20200818-ncs1-rc3-560-g02e61a2ed
-- Found Python3: /usr/bin/python3.9 (found version "3.9.5") found components: Interpreter 
CMake Warning at /var/ncs/zephyr/CMakeLists.txt:764 (message):
  No SOURCES given to Zephyr library: subsys__net__pkt_filter
  Excluding target from build.
CMake Warning at /var/ncs/zephyr/CMakeLists.txt:1687 (message):
  __ASSERT() statements are globally ENABLED
-- Configuring done
-- Generating done
-- Build files have been written to: /var/chip/examples/all-clusters-app/nrfconnect/build
-- west build: building application
[0/956] cd /var/chip && /usr/bin/pytho...e_ncs.py --check --quiet || ( exit 0 )
[1/956] Preparing syscall dependency handling
[26/956] cd /var/chip/examples/all-clu...s/lib/openthread/etc/cmake/print.cmake
OPENTHREAD_CONFIG_ASSERT_ENABLE=1
OPENTHREAD_CONFIG_BACKBONE_ROUTER_DUA_NDPROXYING_ENABLE=0
OPENTHREAD_CONFIG_BACKBONE_ROUTER_MULTICAST_ROUTING_ENABLE=0
OPENTHREAD_CONFIG_ENABLE_BUILTIN_MBEDTLS=0
OPENTHREAD_CONFIG_ENABLE_BUILTIN_MBEDTLS_MANAGEMENT=0
OPENTHREAD_CONFIG_MAC_CSL_AUTO_SYNC_ENABLE=0
OPENTHREAD_CONFIG_DNS_CLIENT_ENABLE=1
OPENTHREAD_CONFIG_ECDSA_ENABLE=1
OPENTHREAD_CONFIG_SRP_CLIENT_ENABLE=1
OPENTHREAD_CONFIG_PING_SENDER_ENABLE=1
OPENTHREAD_SPINEL_CONFIG_RCP_RESTORATION_MAX_COUNT=0
PACKAGE_NAME=OPENTHREAD
OPENTHREAD_CONFIG_THREAD_VERSION=OT_THREAD_VERSION_1_2
OPENTHREAD_CONFIG_NCP_HDLC_ENABLE=1
KERNEL
__ZEPHYR__=1
_FORTIFY_SOURCE=2
BUILD_VERSION=v2.7.99-ncs1
__LINUX_ERRNO_EXTENSIONS__
USE_PARTITION_MANAGER=0
__PROGRAM_START
NRF52840_XXAA
NRF_802154_ECB_PRIORITY=-1
NRF_802154_SWI_PRIORITY=1
NRF_802154_ENCRYPTION_ENABLED=1
NRF_802154_SECURITY_WRITER_ENABLED=1
NRF_802154_IE_WRITER_ENABLED=1
NRF_802154_CCA_CORR_LIMIT_DEFAULT=2
NRF_802154_CCA_CORR_THRESHOLD_DEFAULT=45
NRF_802154_CCA_ED_THRESHOLD_DEFAULT=45
NRF_802154_CCA_MODE_DEFAULT=NRF_RADIO_CCA_MODE_ED
NRF_802154_USE_RAW_API=1
NRF_802154_PENDING_SHORT_ADDRESSES=16
NRF_802154_PENDING_EXTENDED_ADDRESSES=16
NRF_802154_RX_BUFFERS=16
NRF_802154_CSMA_CA_ENABLED=1
NRF_802154_TX_STARTED_NOTIFY_ENABLED=1
NRF_802154_ACK_TIMEOUT_ENABLED=1
NRF_802154_ENCRYPTION_ENABLED=1
NRF_802154_SECURITY_WRITER_ENABLED=1
NRF_802154_IE_WRITER_ENABLED=1
NRF_802154_INTERNAL_RADIO_IRQ_HANDLING=0
MBEDTLS_CONFIG_FILE=nrf-config.h
MBEDTLS_USER_CONFIG_FILE=nrf-config-user.h
[858/956] No configure step for 'chip-gn'
[859/956] Performing build step for 'chip-gn'
Generating compile_commands took 6ms
Done. Made 77 targets from 82 files in 41ms
[366/366] stamp obj/ABS_PATH/var/chip/config/nrfconnect/chip-gn/default.stamp
[879/956] Building CXX object CMakeFil...l-server/access-control-server.cpp.obj
In file included from /var/chip/src/app/ConcreteAttributePath.h:23,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:27,
                 from /var/chip/src/app/clusters/access-control-server/access-control-server.cpp:21:
/var/chip/src/lib/core/Optional.h: In function 'CHIP_ERROR {anonymous}::LogEntryChangedEvent(const chip::Access::AccessControl::Entry&, const chip::Access::SubjectDescriptor&, chip::app::Clusters::AccessControl::ChangeTypeEnum)':
/var/chip/src/lib/core/Optional.h:71:13: warning: '*((void*)& adminPasscodeID +2)' may be used uninitialized in this function [-Wmaybe-uninitialized]
   71 |             new (&mValue.mData) T(other.mValue.mData);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/chip/src/app/clusters/access-control-server/access-control-server.cpp:371:35: note: '*((void*)& adminPasscodeID +2)' was declared here
  371 |     DataModel::Nullable<uint16_t> adminPasscodeID;
      |                                   ^~~~~~~~~~~~~~~
In file included from /var/chip/src/app/ConcreteAttributePath.h:23,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:27,
                 from /var/chip/src/app/clusters/access-control-server/access-control-server.cpp:21:
/var/chip/src/lib/core/Optional.h:71:13: warning: '*((void*)& adminNodeID +8)' may be used uninitialized in this function [-Wmaybe-uninitialized]
   71 |             new (&mValue.mData) T(other.mValue.mData);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/chip/src/app/clusters/access-control-server/access-control-server.cpp:370:39: note: '*((void*)& adminNodeID +8)' was declared here
  370 |     DataModel::Nullable<chip::NodeId> adminNodeID;
      |                                       ^~~~~~~~~~~
[897/956] Building CXX object CMakeFil...net-network-diagnostics-server.cpp.obj
In file included from /var/chip/src/app/EventLoggingTypes.h:24,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:28,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/attributes/Accessors.h:27,
                 from /var/chip/src/app/clusters/ethernet-network-diagnostics-server/ethernet-network-diagnostics-server.cpp:18:
/var/chip/src/lib/core/CHIPTLV.h: In member function 'virtual CHIP_ERROR {anonymous}::EthernetDiagosticsAttrAccess::Read(const chip::app::ConcreteReadAttributePath&, chip::app::AttributeValueEncoder&)':
/var/chip/src/lib/core/CHIPTLV.h:1408:44: warning: '*((void*)& pHYRate +1)' may be used uninitialized in this function [-Wmaybe-uninitialized]
 1408 |         return Put(tag, to_underlying(data));
      |                                            ^
/var/chip/src/app/clusters/ethernet-network-diagnostics-server/ethernet-network-diagnostics-server.cpp:76:41: note: '*((void*)& pHYRate +1)' was declared here
   76 |     Attributes::PHYRate::TypeInfo::Type pHYRate;
      |                                         ^~~~~~~
[928/956] Building CXX object CMakeFil...ifi-network-diagnostics-server.cpp.obj
In file included from /var/chip/src/app/data-model/DecodableList.h:22,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:29,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/attributes/Accessors.h:27,
                 from /var/chip/src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp:18:
/var/chip/src/app/data-model/Encode.h: In member function 'virtual CHIP_ERROR {anonymous}::WiFiDiagosticsAttrAccess::Read(const chip::app::ConcreteReadAttributePath&, chip::app::AttributeValueEncoder&)':
/var/chip/src/app/data-model/Encode.h:42:29: warning: '*((void*)& rssi +1)' may be used uninitialized in this function [-Wmaybe-uninitialized]
   42 |     return writer.Put(tag, x);
      |                             ^
/var/chip/src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp:156:38: note: '*((void*)& rssi +1)' was declared here
  156 |     Attributes::Rssi::TypeInfo::Type rssi;
      |                                      ^~~~
In file included from /var/chip/src/app/data-model/DecodableList.h:22,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:29,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/attributes/Accessors.h:27,
                 from /var/chip/src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp:18:
/var/chip/src/app/data-model/Encode.h:42:29: warning: '*((void*)& channelNumber +2)' may be used uninitialized in this function [-Wmaybe-uninitialized]
   42 |     return writer.Put(tag, x);
      |                             ^
/var/chip/src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp:138:47: note: '*((void*)& channelNumber +2)' was declared here
  138 |     Attributes::ChannelNumber::TypeInfo::Type channelNumber;
      |                                               ^~~~~~~~~~~~~
In file included from /var/chip/src/app/EventLoggingTypes.h:24,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:28,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/attributes/Accessors.h:27,
                 from /var/chip/src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp:18:
/var/chip/src/lib/core/CHIPTLV.h:1408:44: warning: '*((void*)& version +1)' may be used uninitialized in this function [-Wmaybe-uninitialized]
 1408 |         return Put(tag, to_underlying(data));
      |                                            ^
/var/chip/src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp:120:45: note: '*((void*)& version +1)' was declared here
  120 |     Attributes::WiFiVersion::TypeInfo::Type version;
      |                                             ^~~~~~~
In file included from /var/chip/src/app/EventLoggingTypes.h:24,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:28,
                 from /var/chip/zzz_generated/app-common/app-common/zap-generated/attributes/Accessors.h:27,
                 from /var/chip/src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp:18:
/var/chip/src/lib/core/CHIPTLV.h:1408:44: warning: '*((void*)& securityType +1)' may be used uninitialized in this function [-Wmaybe-uninitialized]
 1408 |         return Put(tag, to_underlying(data));
      |                                            ^
/var/chip/src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp:102:46: note: '*((void*)& securityType +1)' was declared here
  102 |     Attributes::SecurityType::TypeInfo::Type securityType;
      |                                              ^~~~~~~~~~~~
[929/956] Building CXX object CMakeFil...-server/window-covering-server.cpp.obj
FAILED: CMakeFiles/app.dir/var/chip/src/app/clusters/window-covering-server/window-covering-server.cpp.obj 
/opt/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-g++ -DBUILD_VERSION=v2.7.99-ncs1 -DCHIP_HAVE_CONFIG_H -DKERNEL -DMBEDTLS_CONFIG_FILE=\"nrf-config.h\" -DMBEDTLS_USER_CONFIG_FILE=\"nrf-config-user.h\" -DNRF52840_XXAA -DNRF_802154_ACK_TIMEOUT_ENABLED=1 -DNRF_802154_CCA_CORR_LIMIT_DEFAULT=2 -DNRF_802154_CCA_CORR_THRESHOLD_DEFAULT=45 -DNRF_802154_CCA_ED_THRESHOLD_DEFAULT=45 -DNRF_802154_CCA_MODE_DEFAULT=NRF_RADIO_CCA_MODE_ED -DNRF_802154_CSMA_CA_ENABLED=1 -DNRF_802154_ECB_PRIORITY=-1 -DNRF_802154_ENCRYPTION_ENABLED=1 -DNRF_802154_IE_WRITER_ENABLED=1 -DNRF_802154_INTERNAL_RADIO_IRQ_HANDLING=0 -DNRF_802154_PENDING_EXTENDED_ADDRESSES=16 -DNRF_802154_PENDING_SHORT_ADDRESSES=16 -DNRF_802154_RX_BUFFERS=16 -DNRF_802154_SECURITY_WRITER_ENABLED=1 -DNRF_802154_SWI_PRIORITY=1 -DNRF_802154_TX_STARTED_NOTIFY_ENABLED=1 -DNRF_802154_USE_RAW_API=1 -DUSE_PARTITION_MANAGER=0 -D_FORTIFY_SOURCE=2 -D__LINUX_ERRNO_EXTENSIONS__ -D__PROGRAM_START -D__ZEPHYR__=1 -I/var/chip/examples/all-clusters-app/nrfconnect/main/include -I/var/chip/examples/all-clusters-app/all-clusters-common/include -I/var/chip/zzz_generated/app-common -I/var/chip/zzz_generated/all-clusters-app -I/var/chip/examples/platform/nrfconnect/util/include -I/var/ncs/nrf/drivers/mpsl/clock_control -I/var/ncs/zephyr/include -I/var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/include/generated -I/var/ncs/zephyr/soc/arm/nordic_nrf/nrf52 -I/var/ncs/zephyr/lib/libc/newlib/include -I/var/ncs/zephyr/lib/util/fnmatch/. -I/var/ncs/zephyr/soc/arm/nordic_nrf/common/. -I/var/ncs/zephyr/subsys/bluetooth -I/var/ncs/zephyr/subsys/net/lib/sockets/. -I/var/ncs/zephyr/subsys/net/lib/openthread/platform/. -I/var/ncs/zephyr/subsys/settings/include -I/var/ncs/nrf/include -I/var/ncs/nrf/lib/multithreading_lock/. -I/var/ncs/nrf/subsys/bluetooth/controller/. -I/var/ncs/zephyr/drivers/flash -I/var/ncs/modules/hal/cmsis/CMSIS/Core/Include -I/var/ncs/modules/hal/nordic/nrfx -I/var/ncs/modules/hal/nordic/nrfx/drivers/include -I/var/ncs/modules/hal/nordic/nrfx/mdk -I/var/ncs/zephyr/modules/hal_nordic/nrfx/. -I/var/ncs/modules/debug/segger/SEGGER -I/var/ncs/modules/debug/segger/Config -I/var/ncs/zephyr/modules/segger/. -I/var/ncs/modules/crypto/tinycrypt/lib/include -I/var/ncs/nrfxlib/nfc/include -I/var/ncs/nrfxlib/mpsl/include -I/var/ncs/nrfxlib/mpsl/include/protocol -I/var/chip/examples/all-clusters-app/nrfconnect/build/modules/nrfxlib/nrfxlib/nrf_security/src/include/generated -I/var/ncs/nrfxlib/nrf_security/include -I/var/ncs/nrfxlib/nrf_security/include/mbedtls -I/var/ncs/nrfxlib/nrf_security/include/psa -I/var/ncs/mbedtls/include -I/var/ncs/mbedtls/include/mbedtls -I/var/ncs/mbedtls/include/psa -I/var/ncs/mbedtls/library -I/var/ncs/nrfxlib/nrf_security/../crypto/nrf_oberon/include -I/var/ncs/nrfxlib/nrf_security/../crypto/nrf_oberon/include/mbedtls -I/var/ncs/nrfxlib/nrf_security/../crypto/nrf_oberon/include/psa -I/var/ncs/nrfxlib/crypto/nrf_oberon/include/mbedtls -I/var/ncs/nrfxlib/crypto/nrf_oberon/include -I/var/ncs/nrfxlib/nrf_802154/driver/include -I/var/ncs/nrfxlib/nrf_802154/sl/include -I/var/ncs/nrfxlib/softdevice_controller/include -I/var/chip/examples/all-clusters-app/nrfconnect/build/modules/openthread/build/etc/cmake -I/var/ncs/modules/lib/openthread/etc/cmake -I/var/ncs/modules/lib/openthread/include -I/var/ncs/zephyr/include/posix -I/var/chip/src -I/var/chip/src/include -I/var/chip/src/lib -I/var/chip/third_party/nlassert/repo/include -I/var/chip/third_party/nlio/repo/include -I/var/chip/examples/all-clusters-app/nrfconnect/build/modules/connectedhomeip/gen/include -isystem /var/ncs/modules/lib/openthread/zephyr/../include -isystem /var/ncs/modules/lib/openthread/zephyr/../examples/platforms -isystem /var/ncs/nrfxlib/crypto/nrf_cc310_platform/include -Werror -Wno-error=maybe-uninitialized -Os -fcheck-new -std=c++14 -Wno-register -fno-exceptions -fno-rtti -imacros /var/chip/examples/all-clusters-app/nrfconnect/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfpu=fpv4-sp-d16 -mfloat-abi=hard -mfp16-format=ieee -imacros /var/ncs/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/var/chip/examples/all-clusters-app/nrfconnect=CMAKE_SOURCE_DIR -fmacro-prefix-map=/var/ncs/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/var/ncs=WEST_TOPDIR -ffunction-sections -fdata-sections -specs=nano.specs -std=gnu++17 -D_SYS__PTHREADTYPES_H_ -MD -MT CMakeFiles/app.dir/var/chip/src/app/clusters/window-covering-server/window-covering-server.cpp.obj -MF CMakeFiles/app.dir/var/chip/src/app/clusters/window-covering-server/window-covering-server.cpp.obj.d -o CMakeFiles/app.dir/var/chip/src/app/clusters/window-covering-server/window-covering-server.cpp.obj -c /var/chip/src/app/clusters/window-covering-server/window-covering-server.cpp
In file included from /var/chip/src/app/clusters/window-covering-server/window-covering-server.cpp:18:
/var/chip/src/app/clusters/window-covering-server/window-covering-server.h:72:22: error: 'chip::app::Clusters::WindowCovering::OperationalStatus::global' is too small to hold all values of 'enum class chip::app::Clusters::WindowCovering::OperationalState' [-Werror]
   72 |     OperationalState global : 2; // bit 0-1 M
      |                      ^~~~~~
/var/chip/src/app/clusters/window-covering-server/window-covering-server.h:73:22: error: 'chip::app::Clusters::WindowCovering::OperationalStatus::lift' is too small to hold all values of 'enum class chip::app::Clusters::WindowCovering::OperationalState' [-Werror]
   73 |     OperationalState lift : 2;   // bit 2-3 LF
      |                      ^~~~
/var/chip/src/app/clusters/window-covering-server/window-covering-server.h:74:22: error: 'chip::app::Clusters::WindowCovering::OperationalStatus::tilt' is too small to hold all values of 'enum class chip::app::Clusters::WindowCovering::OperationalState' [-Werror]
   74 |     OperationalState tilt : 2;   // bit 4-5 TL
      |                      ^~~~
cc1plus: all warnings being treated as errors
[938/956] Building CXX object CMakeFil.../zap-generated/cluster-objects.cpp.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/local/bin/cmake --build /var/chip/examples/all-clusters-app/nrfconnect/build

--- a/src/app/clusters/window-covering-server/window-covering-server.h
+++ b/src/app/clusters/window-covering-server/window-covering-server.h
@@ -67,13 +67,13 @@ enum class OperationalState : uint8_t
 };
 static_assert(sizeof(OperationalState) == sizeof(uint8_t), "OperationalState Size is not correct");
 
+// Decoded components of the OperationalStatus attribute
 struct OperationalStatus
 {
-    OperationalState global : 2; // bit 0-1 M
-    OperationalState lift : 2;   // bit 2-3 LF
-    OperationalState tilt : 2;   // bit 4-5 TL
+    OperationalState global; // bit 0-1 M
+    OperationalState lift;   // bit 2-3 LF
+    OperationalState tilt;   // bit 4-5 TL
 };
-static_assert(sizeof(OperationalStatus) == sizeof(uint8_t), "OperationalStatus Size is not correct");
 
 struct SafetyStatus
 {

